OpenRA.Mods.Common/AssetBrowserFileEndingsFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA
+{
+	public class AssetBrowserFileEndingsFilter : IGlobalModData
```

Comment:
Just `AssetBrowser` would be better. We don't gain anything by making the name more specific, and it prevents us from adding other asset browser related things here in the future.